Source/Demo/CurveTableRowFloatExtractor.cpp

```diff
@@ -0,0 +1,6 @@
+#include "CurveTableRowFloatExtractor.h"
+
+float UCurveTableRowFloatExtractor::GetValue_Implementation() 
+{
+	return RowHandle.Eval(Value, FString(""));
```

Comment:
You'd better use TEXT() macros here and fill it with something:
`Eval(Value, TEXT("CurveTable Float Extractor"))`